src/components/Modals/CreateSingleItemModal/UploadVideoStep/UploadVideoStep.tsx

```diff
@@ -128,15 +147,15 @@ export default class UploadVideoStep extends React.PureComponent<Props, State> {
             </div>
           )}
         </Modal.Content>
-        {video && (
+        {(video || !required) && (
           <Modal.Actions className={styles.actions}>
             <Row grow>
               <Column grow shrink>
                 <Button onClick={this.handleGoBack}>{t('global.back')}</Button>
               </Column>
               <Column align="right">
                 <Button primary onClick={onSaveVideo}>
-                  {t('global.save')}
+                  {required || video ? t('global.save') : t('global.skip')}
```

Comment:
Shouldn't we destroy the video / remove it from the contents record if the user has saved one before and now is skipping the step?

https://www.loom.com/share/00e730dea9674e86b61010dbc3eb7084?sid=a445b66b-a8f5-4b22-858d-c33d889bb942